packages/calcite-components/src/components/handle/handle.tsx

```diff
@@ -298,24 +299,26 @@ export class Handle implements LoadableComponent, T9nComponent, InteractiveCompo
 
   render(): VNode {
     return (
-      // Needs to be a span because of https://github.com/SortableJS/Sortable/issues/1486
-      <span
-        aria-disabled={this.disabled ? toAriaBoolean(this.disabled) : null}
-        aria-label={this.disabled ? null : this.getAriaText("label")}
-        aria-pressed={this.disabled ? null : toAriaBoolean(this.selected)}
-        class={{ [CSS.handle]: true, [CSS.handleSelected]: !this.disabled && this.selected }}
-        onBlur={this.handleBlur}
-        onKeyDown={this.handleKeyDown}
-        role="button"
-        tabIndex={this.disabled ? null : 0}
-        title={this.getTooltip()}
-        // eslint-disable-next-line react/jsx-sort-props -- ref should be last so node attrs/props are in sync (see https://github.com/Esri/calcite-design-system/pull/6530)
-        ref={(el): void => {
-          this.handleButton = el;
-        }}
-      >
-        <calcite-icon icon={ICONS.drag} scale="s" />
-      </span>
+      <Host role="application">
+        <span
+          // Needs to be a span because of https://github.com/SortableJS/Sortable/issues/1486
+          aria-disabled={this.disabled ? toAriaBoolean(this.disabled) : null}
+          aria-label={this.disabled ? null : this.getAriaText("label")}
+          aria-pressed={this.disabled ? null : toAriaBoolean(this.selected)}
+          class={{ [CSS.handle]: true, [CSS.handleSelected]: !this.disabled && this.selected }}
+          onBlur={this.handleBlur}
+          onKeyDown={this.handleKeyDown}
```

Comment:
Based on https://github.com/nvaccess/nvda/issues/7898#issuecomment-529384975, I'm curious if screen readers are triggering a click vs keydown event here. Could you team up w/ @geospatialem to test this out? If it works, it could help avoid the host role.

Reply:
@geospatialem could you review this one?

We don't want to add a click event to this element but we could change its role if we need to. Switching from "button" to "radio" or whatever as suggested by that article.

Reply:
Oh, that's right. There's no click handling logic here. Ignore my suggestion then. Changing the inner element's `role` makes sense.